### ISSUE TITLE
BN can't see through typedef struct

### ISSUE THREAD

**Issue by withzombies** [Vector35/binaryninja-api#1461]
```c
struct Foo {
   int64_t field_0;
   int64_t field_8;
};

typedef struct Foo *LPFOO;
```

![image](https://user-images.githubusercontent.com/4872355/66276877-15682b00-e865-11e9-881b-5a80b606c915.png)

![image](https://user-images.githubusercontent.com/4872355/66276878-19944880-e865-11e9-98bf-1875647a4c8c.png)

**Comment by jeffli678** [comment]
I cannot reproduce this. It works on my end:

![Screenshot from 2020-08-28 18-15-12](https://user-images.githubusercontent.com/3380590/91550033-7fe0f880-e95a-11ea-97b4-a2cfc1818af6.png)

Could you please test it on the latest dev? If it persists, could you please share the binary or copy-and-paste the related assembly lines so I can work on the very same target. 

Btw, which OS are you on?

**Comment by jeffli678** [comment]
This is fixed between dev 2.0.2170 (stable 2.0) and dev 2.1.2263 (stable 2.1), though we do not have a precise build number
